mobileChatExamples/androidChatExample/README.md

```diff
@@ -51,83 +56,159 @@ https://github.com/amazon-connect/amazon-connect-chat-ui-examples/assets/1439784
-
-```mermaid
-flowchart TD
-    style D fill:#fc6b03 size:10
-    A[Start Chat] --> B{Check for participantToken}
-    B -->|Exists| J[Fetch Chat Transcript]
-    B -->|Does not exist| D{Check for contactId}
-    D -->|Exists| E{User Choice}
-    E -->|Restore| F[Use existing contactId]
-    E -->|New Chat| G[Delete stored contactId and participantToken]
-    F --> H[Create new participant connection]
-    G --> I[Start fresh chat session]
-    H --> J[Fetch Chat Transcript]
-    I --> J
-    J --> K[Continue where left off]
+## Implementation
+
+The first step is to call the `StartChatContact` API and pass the response details into the SDK’s `ChatSession` object. Here are some examples of how we would set this up in Kotlin. For reference, you can visit the `androidChatExample` demo within the Amazon Connect Chat UI Examples GitHub repository.
+
+### Configuring and Using `ChatSession` in Your Project
+
+The first step to leveraging the Amazon Connect Chat SDK after installation is to import the library into your file. Next, let's call the StartChatContact API and pass the response details into the SDK’s ChatSession object.  Here is an [example](TODO - Add link to UI Example) of how we would set this up in Kotlin. For reference, you can visit the [AndroidChatExample demo](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/mobileChatExamples/androidChatExample) within the [Amazon Connect Chat UI Examples](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master) GitHub repository.
+
+The majority of the SDKs functionality will be accessed through the `ChatSession` object. In order to use this object in the file, you can inject it using `@HiltViewModel`:
+
+```
+class ChatViewModel @Inject constructor(
+    private val chatSession: ChatSession, // Injected ChatSession
+    private val chatRepository: ChatRepository,
+    private val sharedPreferences: SharedPreferences,
+) : ViewModel() {
+```
+
+If you are not using Hilt, then you can initialise `ChatSession` like this:
+
+```
+private val chatSession = ChatSessionProvider.getChatSession(context)
+```
+
+In this example, we are using a `ChatViewModel` class that helps bridge UI and SDK communication.  This class is responsible for managing interactions with the SDK's ChatSession object. From here, we can access the SDK's suite of APIs from the `chatSession` property. 
+
+Before using the chatSession object, we need to set the config for it via the GlobalConfig object.  Most importantly, the GlobalConfig object will be used to set the AWS region that your Connect instance lives in.  Here is an example of how to configure the ChatSession object:
+
+```
+private suspend fun configureChatSession() {
+      val globalConfig = GlobalConfig(region = chatConfiguration.region)
+      chatSession.configure(globalConfig)
+      ...
+  }
+```
+
+From here, you are now ready to interact with the chat via the `ChatSession` object.
+
+#### SendMessage
```

Comment:
When showing these examples of how to use `ChatSession`, you should also refer them to the Android SDK README for full API documentation.